templates/partials/moduleHighlightedCourse.html.twig

```diff
@@ -1,33 +1,33 @@
 {#
  * Template: partials/moduleHighlightedCourse.html.twig
  *
- * @param Contentful\Delivery\DynamicEntry module
+ * @param Contentful\Delivery\Resource\Entry module
  #}
 
-{% set course = module.getCourse() %}
+{% set course = module.course %}
  <div class="module module-highlighted-course">
-    <div class="module-highlighted-course__wrapper" style="background-image: url('{{ course.getImage().getFile().getUrl() }}');">
+    <div class="module-highlighted-course__wrapper" style="background-image: url('{{ course.image.file.url }}');">
```

Comment:
Will this fail gracefully if you point to an unpublished image?

Reply:
I'm... unsure, tbh. What I know is that the underlying behavior didn't change, so it was failing _ungracefully_ before, it will keep doing that now 🤔

Reply:
👍